### DISCUSSION TITLE
How to assign permission into a tenancy

### DISCUSSION THREAD

**Discussion by KatiaSishost** [spatie/laravel-permission#1704] [Q&A]
Hello im using a Hyn with Multi subdomains and 1 single database.

i need add Spatie Permission and it work perfectly... i can difference Users and Admins... but i have a problem... 1 user cant acces to admin area... admin can acces to admin and user areas, without problem...

but... test1.example.com/admin admin can acces to admin panel of test2.example.com/admin...

i need users can enter into test1.example.com and test2.example.com but the admin only can ingress at your admin panel only

how can solve it?

**Comment by fredsal** [comment]
Can you try https://github.com/spatie/laravel-permission/discussions/1782
